check_source_in_factory.py

```diff
@@ -148,54 +148,59 @@ def _check_project(self, project, package, rev):
     def _check_requests(self, project, package, rev):
         self.logger.debug("checking requests")
         prjprefix = ''
+        apiurl = self.apiurl
+        sr = 'sr'
         try:
-            apiurl = self.apiurl
             if self.config.project_namespace_api_map:
-                for prefix, url in self.config.project_namespace_api_map:
+                for prefix, url, srprefix in self.config.project_namespace_api_map:
                     if project.startswith(prefix):
                         apiurl = url
                         prjprefix = prefix
                         project = project[len(prefix):]
+                        sr = srprefix
                         break
             requests = osc.core.get_request_list(apiurl, project, package, None, ['new', 'review'], 'submit')
         except (urllib2.HTTPError, urllib2.URLError):
             self.logger.error("caught exception while checking %s/%s", project, package)
             return None
 
+        def srlink(reqid):
```

Comment:
nit: Perhaps `srref` or `sr_ref` or similar since it returns reference notation and link notation.